bundles/framework/printout/view/BasicPrintout.js

```diff
@@ -770,6 +778,16 @@ Oskari.clazz.define('Oskari.mapframework.bundle.printout.view.BasicPrintout',
             if (selections.scaleText) {
                 url = url + '&scaleText=' + selections.scaleText;
             }
+
+            if (me.areLayersWithTimeSeriesSelected) {
```

Comment:
Do this again instead of using an instance variable:

     me.instance.sandbox.findAllSelectedMapLayers()
                    .filter(l => l.getAttributes().times).length > 0;

Reply:
Add a check for timeseriesPlugin not being undefined as well.

Reply:
Fixed as proposed.